tcpdf.php

```diff
@@ -7866,6 +7891,60 @@ public function _destroy($destroyall=false, $preserve_objcopy=false) {
 		}
 	}
 
+	/** Page cache reference counts */
+	protected static $pageCacheRefCnts = [];
```

Comment:
```suggestion
	protected static $pageCacheRefCnts = array();
```
Old php compatibility